_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -213,8 +213,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    const context = this;
+    _.forIn(objectData, (value, key) => {
+      if (value === null) {
+        _.set(objectData, key, '');
+      } else if (typeof (value) === 'object') {
+        context.resetNullValues(value);
+      }
+    });
+
+    return objectData;
```

Comment:
Hmm...this API both mutates the parameter and passes it back as if it were an expression (which would imply it returns a new copy rather than changing the parameter).

Also, mutating the object while iterating over it is always a little bit weird feeling.  I would recommend using _.mapValues (https://lodash.com/docs#mapValues) to handle this. That way you don't mutate the object that you are visiting.